run_dir/static/js/running_notes.js

```diff
@@ -100,13 +100,23 @@ function load_running_notes(wait) {
   // Clear previously loaded notes, if so
   note_url = get_note_url();
   $("#running_notes_panels").empty();
+  // From agreements tab
+  $("#invoicing_notes").empty();
+  inv_notes = {};
   return $.getJSON(note_url, function(data) {
     if(Object.keys(data).length == 0 || typeof data === 'undefined'){
       $('#running_notes_panels').html('<div class="well">No running notes found.</div>');
     } else {
       $.each(data, function(date, note) {
         $('#running_notes_panels').append(make_running_note(date, note));
+        if(note['category'].includes('Invoicing')){
+          Object.assign(inv_notes, { date: date, note: note });
+          $('#invoicing_notes').append(make_running_note(inv_notes.date, inv_notes.note, true));
```

Comment:
Huh, if you do it like this, you don't need to save the notes in `inv_notes` at all, just do 
```suggestion
          $('#invoicing_notes').append(make_running_note(date, note, true));
```
Just make it a boolean which is true if there are any invoicing notes and then you can check if it's true or false below on line 117. You don't need line 113 either.